samza-api/src/main/java/org/apache/samza/job/model/ContainerModel.java

```diff
@@ -54,6 +58,15 @@ public String getId() {
     return tasks;
   }
 
+  /**
+   * Returns the {@link SystemStreamPartition}s that this container is responsible for consuming.
+   * @return {@link SystemStreamPartition}s for this container
+   */
+  public Set<SystemStreamPartition> getSystemStreamPartitions() {
```

Comment:
For consistency with the `JobModel`, getters ignored in json serde should probably be moved to a util class similar to `JobModelUtil`